modules/virt-configuring-storage-class-bootsource-update.adoc

```diff
@@ -39,9 +39,13 @@ spec:
       template:
         spec:
           storageClassName: <new_storage_class> <1>
+      schedule: "0 */12 * * *" <2>
+      managedDataSource: <data_source> <3>
 #...
 ----
 <1> Define the storage class.
+<2> Required: Schedule for the job specified in cron format.
+<3> Required: For the custom image to be detected as an available boot source, the name of the image's `managedDataSource` must match the name of the template's `DataSource`, which is found under `spec.dataVolumeTemplates.spec.sourceRef.name` in the VM template YAML file.
```

Comment:
**QE REVIEW NEEDED** Please verify that these parameters are required and the definitions are correct.